network-api/networkapi/management/commands/migrate_cloudinary.py

```diff
@@ -0,0 +1,50 @@
+import ntpath
+import requests
+
+from django.core.management.base import BaseCommand
+
+from django.core.files.images import ImageFile
+from io import BytesIO
+from mimetypes import MimeTypes
+from PIL import Image as PILImage
+
+from wagtail.images.models import Image as WagtailImage
+from networkapi.wagtailpages.pagemodels.products import ProductPage
+
+
+class Command(BaseCommand):
+    help = 'Migrate PNI Cloudinary images to Wagtail images'
+
+    def handle(self, *args, **options):
+
+        for product in ProductPage.objects.all():
+            if product.cloudinary_image:
```

Comment:
let's add a little `print` here that tells us which product we're in (like "Processing product 23 of 317") so that running it via ssh keeps progress visible.

Reply:
Done!